### PR TITLE
Fix escape dobule quote

### PR DESCRIPTION
**Pull request by heavenshell**
Sorry my previous PR ( #2 ) was incompleted.
I should escape double quote.

https://github.com/thomasfaingnaert/vim-lsp-ultisnips/blob/master/autoload/lsp_ultisnips.vim#L3

Thx @hrsh7th for review.

### CODE REVIEW

File: autoload/lsp_neosnippet.vim [hunk 1]
```diff
@@ -12,5 +13,5 @@ function! s:escape_snippet(text) abort
 endfunction
 
 function! lsp_neosnippet#expand_snippet(params) abort
-    call feedkeys("\<C-r>=neosnippet#anonymous('" . s:escape_snippet(a:params.snippet) . "')\<CR>")
+    call feedkeys("\<C-r>=neosnippet#anonymous(\"" . s:escape_snippet(a:params.snippet) . "\")\<CR>")
```
Comment (heavenshell) [inline autoload/lsp_neosnippet.vim:16]: Revert.

**Comment by thomasfaingnaert** [comment]
Hmm, is there something that wasn't working with #2? I personally like that, because it also handles `\` and other special characters.

**Comment by heavenshell** [comment]
Thank you for comment.

Currently no. but if `a:text` contains single quoted string(something like `'name': '$1'`), `E116: Invalid arguments` will raise.
So, IMO escaping `a:text` is better.

**Comment by thomasfaingnaert** [comment]
True, but would doing a substitute of `'` to `''` not fix that. I think that's better, since it handles all special characters.

**Comment by heavenshell** [comment]
>True, but would doing a substitute of ' to '' not fix that. I think that's better, since it handles all special characters.

humm, what should we do?
If PR #2 is enough, it's ok to close this PR.
I think single quote contained text are not coming right now.

**Comment by thomasfaingnaert** [comment]
I was thinking something like #4. Would that not work in all cases?

**Comment by heavenshell** [comment]
#4 is LGTM!

**Comment by thomasfaingnaert** [comment]
I've merged #4. Thanks for your feedback!

**Pull request closed without merging**